Lex-Sample/Swift/LexSwift/VoiceChatViewController.swift

```diff
@@ -24,22 +24,21 @@ class VoiceChatViewController: UIViewController, AWSLexVoiceButtonDelegate {
     @IBOutlet weak var input: UILabel!
     @IBOutlet weak var output: UILabel!
     
-    
     override func viewDidLoad() {
         super.viewDidLoad()
         (self.voiceButton as AWSLexVoiceButton).delegate = self
     }
     
     func voiceButton(_ button: AWSLexVoiceButton, on response: AWSLexVoiceButtonResponse) {
         DispatchQueue.main.async(execute: {
-            // `inputranscript` is the transcript of the voice input to the operation
+            // `inputTranscript` is the transcript of the voice input to the operation
             if let inputTranscript = response.inputTranscript {
                 print("Input Transcript: " + inputTranscript)
                 self.input.text = "\"\(inputTranscript)\""
             }
             
             if let outputText = response.outputText {
-                print("Input Transcript: " + outputText)
+                print("Output Transcript: " + outputText)
             }
 
             self.output.text = response.outputText
```

Comment:
this looks kind of odd, that `self.output.text` is being set to `response.outputText` and could be nil? what happens if self.out.text to nil? If there is a possible crash here, can we update this?

```swift
if let outputText = response.outputText {
  self.output.text = outputText
} else {
  self.output.text = ""
}
```